### PR TITLE
feat(currency): Currency history interface

### PR DESCRIPTION
**Pull request by gabriele-v**
First step to complete #306 

UI is complete, better to merge it now because all logic will require more time maybe creating conflicts difficoult to solve.

### CODE REVIEW

File: src/currencydialog.cpp [hunk 1]
```diff
@@ -231,59 +206,65 @@ void mmCurrencyDialog::CreateControls()
     wxBoxSizer* itemBoxSizer22 = new wxBoxSizer(wxHORIZONTAL);
     itemBoxSizer2->Add(itemBoxSizer22, wxSizerFlags(g_flags).Centre());
 
-    wxButton* itemButton24 = new wxButton( this, wxID_REFRESH, _("&Update"));
+    wxButton* itemButton24 = new wxButton(this, wxID_OK, _("&OK"));
```
Comment (vomikan) [inline src/currencydialog.cpp]: _("&OK") should be _("&OK ")

Comment (gabriele-v) [inline src/currencydialog.cpp]: Fixed in my trunk (cannot commit because unstable)

**Comment by vomikan** [comment]
Don't forget put into changed files
Copyright (C) 2015 Gabriele-V

**Comment by vomikan** [comment]
Suggesions:
1. After "Online Update" the currency history list should be refreshed.
2. Add "Delete history for unused currencies" function.

**Pull request merged**